### DISCUSSION TITLE
Problem with App Published (2 splash screen and white screen with capacitor)

### DISCUSSION THREAD

**Discussion by w3ba1809** [quasarframework/quasar#12479] [CLI - Capacitor / Cordova modes]
Hello, I need your help with an issue that I have with an app made with capacitor:

-Context: when I run the app in dev mode all works well, the problem is when I upload the app to the app store o play store I create an internal test and send the link to the testers when they installed the app occurs these issues:

1. The app shows 2 splash screens one with grey background and logo and after the other with white background and logo, in dev mode I don't see this.

2. After the splash screens are shown the app show a white screen and stops there no login screen is shown.

Please any help is welcome I am stuck with this issue.

Attach a video with both issues.  

Thanks for any help.

https://user-images.githubusercontent.com/46602589/153942129-9ee51645-4b56-45fc-8983-5d8c5c0dfb94.mp4

**Comment by 3m1n3nc3** [comment]
Did you figure out a solution to this issue? I'm expereincing the same.
